### PR TITLE
Fix opened tables gauge => derive

### PR DESCRIPTION
**Pull request by jeroenvermeulen**
The number of opened tables was ever increasing while it would look a lot better to show only changes. I fixed it by changing removing the GAUGE type from the graph level of 'Tables' and adding it to the fields that are indeed a GAUGE. Also I added field explanations.
Old situation:
mysql_tables-week

### CODE REVIEW

**Comment by roidelapluie** [comment]
Thanks. You need to adapt the tests too.

File: lib/Munin/MySQL/Graph/Standard.pm [hunk 1]
```diff
@@ -155,26 +155,31 @@ sub graphs { return {
                 title  => 'Tables',
             },
             data_source_attrs => {
-                type  => 'GAUGE',
                 draw  => 'LINE1',
             },
         },
         data_sources => [
             {name => 'table_open_cache', label  => 'Table cache',
+                                         type   => 'GAUGE',
```
Comment (roidelapluie) [inline lib/Munin/MySQL/Graph/Standard.pm]: is it needed for the table cache too?

Comment (jeroenvermeulen) [inline lib/Munin/MySQL/Graph/Standard.pm]: There I am just keeping the setting which was previously set on table level.

**Comment by jeroenvermeulen** [comment]
Another small change is comming, changing the colour + area to line1 of the `Innodb Table Cache Limit`

**Comment by jeroenvermeulen** [comment]
I changed the colour + type of 'Innodb Table Cache Limit' because the area made the 'Open tables' line hardly visible + overlapped with the 'Table cache' area.
New look:
mysql_tables-day

**Comment by roidelapluie** [comment]
Thanks!!

**Pull request merged**